src/scancode/plugin_only_findings.py

```diff
@@ -49,6 +49,8 @@ def process_only_findings(active_scans, results):
 def has_findings(active_scans, scanned_file):
     """
     Return True if the `scanned_file` has findings for any of the
-    `active_scans` names list (excluding basic file information).
+    `active_scans` names list (excluding basic file information)
+    or any errors occured when scanning the file.
     """
-    return any(scanned_file.get(scan_name) for scan_name in active_scans)
+    findings = active_scans + ['scan_errors']
```

Comment:
That's a sleek and simple way to include this, even though this is not a "scan" :+1:

Reply:
Thanks. First I looked into adding it to `active_scans`, but it doesn't really belong there and I did not want to make changes that affect other plugins.